apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -424,6 +453,14 @@ export default class TutorialExplorer extends React.Component {
           return false;
         }
 
+        if (
+          searchTerm !== '' &&
+          (!tutorial.name.toLowerCase().includes(searchTerm) ||
+            !tutorial.longdescription.toLowerCase().includes(searchTerm))
```

Comment:
I noticed "oceans" wasn't showing any results.  Turns out the current logic requires both name and description have the string.  A simple change to this seems to fix it:

``` if (
          searchTerm !== '' &&
          !(tutorial.name.toLowerCase().includes(searchTerm) ||
            tutorial.longdescription.toLowerCase().includes(searchTerm))
        ) {
```

It might also be good to trim whitespace from the beginning and end of the string.

Reply:
thanks for catching!